spec/fixtures/iptables/conversion_hash.rb

```diff
@@ -255,10 +255,10 @@
     },
   },
   'string_escape_sequences' => {
-    line: '-A INPUT -m comment --comment "000 parse escaped \\"s, \\\'s, and \\\\s"',
+    line: '-A INPUT -m comment --comment "000 parse escaped \\"s, \\"s, \\\'s, \\\'s, \\\\s and \\\\s"',
     table: 'filter',
     params: {
-      name: '000 parse escaped "s, \'s, and \\s',
+      name: '000 parse escaped "s, "s, \'s, \'s, \\s and \\s',
```

Comment:
Why did you double up each of these?

Reply:
Ah, I'm an idiot. Need to stop reviewing first thing on a monday. Will make an adhoc run, and if it passes will merge.

Reply:
Haha, cheers!